Remove pushers when deleting 3pid from account (#10581)
matrix-org/synapse: CHANGES.md

> Users will stop receiving message updates via email for addresses that were once, but not still, linked to their account

Reviewer: ```suggestion
Users will stop receiving message updates via email for addresses that were previously linked to their account.
```

Might be slightly simpler wording.
